README.md

```diff
@@ -43,12 +43,16 @@ us manage and test the application.
 * We get the tools we depend upon via `npm`, the [node package manager][npm].
 * We get the Angular code via `bower`, a [client-side code package manager][bower].
 
+Switch to the Guest network for the install steps in order to access the necessary repositories.
```

Comment:
Since this is technically an open-source project, I feel like this is a weird thing to include in the readme.

Additionally, if you have the new registry configured, this step shouldn't even be necessary internally.